duktape.js

```diff
@@ -139,3 +144,76 @@ fs.readdirSync('.').forEach(function (filename) {
 
 console.log(testCount + ' tests executed: ' + testSuccess + ' success, ' + (testCount - testSuccess) + ' fail');
 console.log(testOutOfDate + ' tests are out of date (data-*.js file .res)');
+
+console.log("Writing New Results To File...");
+for (var i = 0; i < fixTests.length; i++) {
+    var path = fixTests[i].path;
+    var success = fixTests[i].success;
+    var data = fs.readFileSync(path[0] + ".js", "utf8").split("\n");
+    var k = 1;
+    var done = false;
+    var line = 0;
+
+    // Find Test
+    while (!done) {
+        if (data[line].includes("'" + path[k].replace(new RegExp(/'/g/, "g"), "\\'") + "'")) {
```

Comment:
sorry i wasn't clear; i mean that use of `new RegExp` is bad and should be avoided whenever possible in javascript. This can just be:
```suggestion
        if (data[line].includes("'" + path[k].replace(/'/g, "\\'") + "'")) {
```

Reply:
Already Done!

Reply:
It's not done in your latest version; perhaps there's a typo?